src/widgets/MainTask/MainTask.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+    render, screen, fireEvent, waitFor,
+} from '@testing-library/react';
+import { testTask } from 'shared/test/TestTask';
+import { setupStore } from 'app/store';
+import { Provider } from 'react-redux';
+import { getDateAndTime } from 'shared/lib/helpers';
+import { MainTask } from './MainTask';
+
+import '@testing-library/jest-dom/extend-expect';
+
+jest.mock('shared/lib/helpers');
+
+const mockTask = testTask;
```

Comment:
not sure we really need to have a separate const here

Reply:
Fixed